src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,42 @@
 package core.basesyntax;
 
+abstract class Machine {
+    public abstract void doWork();
+    public abstract void stopWork();
+}
+class Truck extends Machine {
+    public void doWork() {
+        System.out.println("Truck start");
+    }
+    public void stopWork() {
+        System.out.println("Truck stop");
+    }
+}
+class Bulldozer extends Machine {
+    public void doWork() {
+        System.out.println("Bulldozer start");
+    }
+
+    public void stopWork() {
+        System.out.println("Bulldozer stop");
```

Comment:
The message 'Bulldozer stop' should be more descriptive. For instance, 'Bulldozer has stopped working'.